presto-jmx/src/main/java/io/prestosql/plugin/jmx/JmxTableHandle.java

```diff
@@ -35,18 +37,21 @@
     private final List<String> objectNames;
     private final List<JmxColumnHandle> columnHandles;
     private final boolean liveData;
+    private final TupleDomain<ColumnHandle> nodeFilter;
 
     @JsonCreator
     public JmxTableHandle(
             @JsonProperty("tableName") SchemaTableName tableName,
             @JsonProperty("objectNames") List<String> objectNames,
             @JsonProperty("columnHandles") List<JmxColumnHandle> columnHandles,
-            @JsonProperty("liveData") boolean liveData)
+            @JsonProperty("liveData") boolean liveData,
+            @JsonProperty("constraint") TupleDomain<ColumnHandle> nodeFilter)
     {
         this.tableName = requireNonNull(tableName, "tableName is null");
         this.objectNames = ImmutableList.copyOf(requireNonNull(objectNames, "objectName is null"));
         this.columnHandles = ImmutableList.copyOf(requireNonNull(columnHandles, "columnHandles is null"));
         this.liveData = liveData;
+        this.nodeFilter = requireNonNull(nodeFilter, "constraint is null");
```

Comment:
I guess `nodeFilter is null` is expected message and L48 should be updated `@JsonProperty("nodeFilter")`.

Reply:
Oh, that's true!